ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -248,3 +258,15 @@ if ! wait "$!"; then
   # TODO: gather logs??
   exit 1
 fi
+
+# Exit normally if there is no day2 jobs
```

Comment:
We don't need this as it's the same content as the other worker.ign file you're overwriting. You find this procedure in the doc because users might od this a lot of time later the cluster installation

Reply:
@aleskandro I think there is no the other worker.ign file for agent, if you don't create it manually.

Reply:
Right, thanks for clarifying.